tests/fixtures/FV3_GFS_v16.yaml

```diff
@@ -0,0 +1,45 @@
+sphum:
+  longname: specific humidity
+  units: kg/kg
+  profile_type: fixed
+  surface_value: 1.e30
```

Comment:
Given the description of how profile_type is specified, I think it might make the most sense to next `surface_value` under `profile_type`, and to iterate through each of the entries under `profile_type`. That will help us ensure the that we can handle all options potentially provided by this file format. I suppose that means, you'd also want to store the method `name` in the nest as well.

Reply:
Nesting updated, will ensure method name options are included in FieldTableConfig as well